<!-- 
  record servo positions
-->


<launch>

  <node pkg="sheldon_servos" name="record_servos" type="record_servos.py" output="screen">
  </node>
</launch>
